pkg/selinux/setroubleshoot.js

```diff
@@ -140,7 +140,7 @@ const initStore = function(rootElement) {
                     dataStore.render();
                 })
                 .catch(error => {
-                    dataStore.error = error;
+                    dataStore.error = error.toString();
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18213-20230120-140033-28d772ae-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)